assets/src/block-editor/blocks/amp-latest-stories/edit.js

```diff
@@ -50,47 +50,45 @@ class LatestStoriesEdit extends Component {
 		const { attributes, setAttributes, latestStories } = this.props;
 		const { order, orderBy, storiesToShow } = attributes;
 
-		const inspectorControls = (
```

Comment:
Good idea to simply output this in the return statement instead of storing it in a `const`.